openapi/components/schemas/GatewayAccountConfig/Clearhaus.yaml

```
@@ -16,4 +16,25 @@ allOf:
         required:
           - apiKey
       threeDSecureServer:
-        $ref: ../ThreeDSecureServers/Clearhaus3dsServers/Clearhaus3dsServers.yaml
+        type: object
+        required:
+          - name
+          - merchantId
+          - acquirerBin
+          - apiKey
+        properties:
+          name:
+            type: string
+            description: Merchant plug-in name.
+            enum:
+              - Clearhaus3dsServer
+          merchantId:
+            type: string
+            description: Merchant account ID or Card Acceptor ID provided by the acquiring bank.
+          acquirerBin:
+            type: string
+            description: Acquiring institution identification code.
```

suggestion: suggestion for readability

(Series of nouns can be difficult to read)

```suggestion
            description: Identification code of the acquiring institution.
```